src/main/java/com/cascadebot/cascadebot/moderation/ModerationManager.java

```diff
@@ -5,8 +5,11 @@
 
 package com.cascadebot.cascadebot.moderation;
 
+import com.cascadebot.cascadebot.CascadeBot;
 import com.cascadebot.cascadebot.commandmeta.CommandContext;
 import com.cascadebot.cascadebot.messaging.MessagingObjects;
+import com.cascadebot.cascadebot.utils.DiscordUtils;
+import net.dv8tion.jda.bot.sharding.ShardManager;
```

Comment:
Can you run Optimise imports in here, to check if there are needed?

Reply:
removed unused imports